Servers/Administration/OJS.Servers.Administration/Controllers/SubmissionTypesController.cs

```diff
@@ -29,6 +33,17 @@ public class SubmissionTypesController : BaseAdminApiController<SubmissionType,
     public async Task<IActionResult> GetForProblem()
         => this.Ok(await this.submissionTypesBusinessService.GetForProblem());
 
+    [HttpPost]
+    [Authorize(Roles = Administrator)]
+    [Authorize(Roles = Developer)]
```

Comment:
Only Developer role should be allowed. Regular Administrator should not be able to perform this action.

Reply:
This endpoint will not be reachable without the Administrator role because the Admin server will cut off the request before we have a chance to get to the endpoint, also both Administrator and Developer roles will be validated for this endpoint meaning an administrator without developer role will not be able to access the endpoint.